velox/exec/RowContainer.cpp

```diff
@@ -297,6 +297,12 @@ char* RowContainer::initializeRow(char* row, bool reuse) {
   return row;
 }
 
+void RowContainer::initializeFields(char* row) {
+  if (row != nullptr) {
```

Comment:
Why this check? Should we not require that 'row' is not null?

Reply:
Per my understanding:
- Given `row` is not required to be non-null, it might be nullptr, meaning that `row` is NULL, then there is no need to zero it.
- We should not pass a nullptr to `memset` since `memset` a nullptr is UB.

Reply:
My impression that initializeFields will be used only after initializeRow which cannot return null, no?

Reply:
It should only be used that way.
This change is added after the request to "remove FOLLY_NONNULL". Then I think I should prevent the caller from misusing the method, so added this.

Reply:
FOLLY_NONNULL is no longer needed because it doesn't do anything. You can add a VELOX_CHECK_NOT_NULL(row) or just assume that row is not null.

Reply:
Oh I misunderstood... Thanks for the explanation. Fixed.